pkg/api/cortex-ruler.go

```diff
@@ -96,7 +96,8 @@ type NamespaceConfigResponse map[string][]GettableRuleGroupConfig
 
 // swagger:model
 type PostableRuleGroupConfig struct {
-	Name     string                     `yaml:"name" json:"name"`
+	Name string `yaml:"name" json:"name"`
+	// Example 1m
```

Comment:
I'm not sure we need these changes here. The same applies for the RuleNode down below.